src/Smalot/PdfParser/Header.php

```diff
@@ -176,14 +161,7 @@ protected function resolveXRef($name)
         return $this->elements[$name];
     }
 
-    /**
-     * @param string   $content  The content to parse
-     * @param Document $document The document
-     * @param int      $position The new position of the cursor after parsing
```

Comment:
Shouldn't this comment be kept?

Reply:
Must have been too eager in removing these. Re-added them.